app/src/main/java/ch/sdp/vibester/BuzzerScoreUpdater.kt

```diff
@@ -24,15 +27,31 @@ class BuzzerScoreUpdater(ids: ArrayList<Int>, scores: Array<Int>) {
         return buzzerToScoreMap
     }
 
+    fun getWinnerId(): Int {
+        return winnerId

```

Comment:
Codeclimate shows this line is not tested. This return should not be hard to test, right? Just calling the function and comparing to the actual result?